.gitignore

```diff
@@ -26,10 +26,13 @@ config/secrets.yml
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
```

Comment:
Add also this to ignore all .DS_Store files:

```
.DS_Store
```
Also remove all of them from this repo: http://stackoverflow.com/a/2047477

more info: http://en.wikipedia.org/wiki/.DS_Store